bandpass/README.md

```diff
@@ -263,17 +329,35 @@ This plot shows a problem with this implementation of a low-pass
 filter: namely, as the critical frequency decreases the system has
 more "inertia" responding to changes in the signal more slowly.  The
 curve gets smoother, but part of that smoothness comes at a cost in
-delay.
+delay of the signal representing the data.
 
 
-<a id="orgfb758bf"></a>
+<a id="orgb4c7aff"></a>
```

Comment:
Looks like the TOC links in this part of the readme are now broken with these ID changes.